src/meta/override_values.yaml

```diff
@@ -0,0 +1,22 @@
+# Missing roots due to circular references:
+minecraft:raw_iron: 100
+minecraft:iron_ingot: 100
```

Comment:
This seems way too high. I got a few stacks easily, so this would give me an insane amount of GRK..